attractions/text-field/text-field.scss

```diff
@@ -119,14 +119,13 @@
 
     label {
       font-family: vars.$font;
-      background-color: #fff;
+      background-color: vars.$textfield-outline-label-bg;
       color: vars.$textfield-border;
       padding: 0 0.25em;
       position: absolute;
       top: 0;
       left: 1.4em;
       transform: translate(-0.25em, -55%);
-      border-radius: 0.3125em;
```

Comment:
Why'd you remove this? The reason why we had it in the first place was, for example, Chrome autocompletion when it gives a bluish tint to the background of the field. That bluish background exposed the rough edges of the label and it wasn't looking too good.

Reply:
Sorry, that was a mistake! Added back.